sqllineage/core/parser/sqlfluff/extractors/select.py

```diff
@@ -150,10 +160,12 @@ def _handle_set(self, segment: BaseSegment) -> None:
                     subquery, alias = sq
                     table_identifier = find_table_identifier(subquery)
                     if table_identifier:
-                        read_sq = SqlFluffTable.of(table_identifier, alias)
                         for seg in list_child_segments(subquery):
+                            for set_subquery in self.list_subquery(seg):
+                                self.subqueries_within_set.append(set_subquery)
+
+                            self._handle_table(seg, holder)
                             self._handle_column(seg)
```

Comment:
Let's remove the `list_subqueries` call. We can get children segment of type `select_statement` and `bracketed` from set_expression. And then list_child_segments from `select_statement` and `bracketed` and feed it to `_handle_table` and `_handle_column`.  We don't need subqueries_within_set that way.

Reply:
I don't quite understand. Can you help me by directly modifying my code?